### ISSUE TITLE
RSKimageCropper issue

### ISSUE THREAD

**Issue by iOSHw** [ivpusic/react-native-image-crop-picker#281]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.12.8
- react-native v0.42.0

### Platform
Tell us to which platform this issue is related

- iOS

### Expected behaviour
run issue   xcode 8.2 has no problem ,when i run in # xcode 8.3 ,it make problem
 

### Actual behaviour
Redefinition of module 'RSKImageCropper'   issue

### Steps to reproduce

1.

2.

3.

### Attachments

// stacktrace or any other useful debug info

**Comment by iOSHw** [comment]
@ivpusic

**Comment by darrenchiu** [comment]
+1

**Comment by springwq** [comment]
The same issue found, Xcode 8.3, macOS 10.12.4

**Comment by juxiaowei** [comment]
+1

**Comment by juxiaowei** [comment]
+1
seems this happened after updating xcode and macos today

**Comment by iOSHw** [comment]
@ivpusic help！

**Comment by habovh** [comment]
Can confirm this is happening on Xcode 8.3, it is unrelated to the macOS update (I'm still on 10.12.3).
It seems that Xcode the `module.modulemap` exists both under `node_modules` folder and in Xcode's derived data folder.

See the path of the files here:
![capture d ecran 2017-03-28 a 13 56 52](https://cloud.githubusercontent.com/assets/4186230/24403811/da9d61ac-13be-11e7-9257-8090062556d7.png)

**Comment by eiosca** [comment]
There is un update here, i'm trying to switch to cocoapods

https://github.com/ruslanskorb/RSKImageCropper/commit/7d5ea88c7e6fd7840cff598cc4cf7d9aafd027ac

**Comment by ivpusic** [comment]
@iOSHw please try commit from @eiosca

**Comment by aforty** [comment]
Anything us non-cocoapods users can do? I'm not about to introduce cocoapods into my project when I was just able to finally get rid of it. Nothing but pain and suffering that thing...

**Comment by iOSHw** [comment]
@ivpusic what time will you fix this issue ?

**Comment by NewBen001** [comment]
@eiosca  Did you fix this problem ?

**Comment by eiosca** [comment]
@NewBen001  yes, but with the bad solution.
Just following the message's commit of https://github.com/ruslanskorb/RSKImageCropper/commit/7d5ea88c7e6fd7840cff598cc4cf7d9aafd027ac

Go to folder 

> node_modules/react-native-image-crop-picker/ios

rename the file 

> module.modulemap > RSKImageCropper.modulemap

then edit the file RSKImageCropper.xcodeproj/project.pbxproj
and add this line two times (in debug and release part Cf Commit):
`MODULEMAP_FILE = "$(SRCROOT)/RSKImageCropper/RSKImageCropper.modulemap";`

Clean and build. It should be works.

PS: It was a fail to me to switch on cocoapods, that's why i'm doing that and waiting for the fix.
I will try to work on a PR this week-end but not sure.

**Comment by basilbai** [comment]
This question has affected my job......

**Comment by krailler** [comment]
I have same issue.. :/

**Comment by krailler** [comment]
modification of @eiosca solve issue <3

**Comment by ivpusic** [comment]
@eiosca could you send pull request?

**Comment by eiosca** [comment]
@ivpusic Yeah, like i said i will try this week-end :-)

**Comment by ivpusic** [comment]
if someone can do it even sooner, PR will be welcome

**Comment by magrinj** [comment]
I create a PR: #282, thanks to @eiosca!
I also try to connect the pod install directly to the `RSKImageCropper` like that:

`
pod 'RSKImageCropper', :git => 'https://github.com/ruslanskorb/RSKImageCropper.git', :branch => 'master'
`

but not working for me :/

**Comment by ivpusic** [comment]
released in https://github.com/ivpusic/react-native-image-crop-picker/releases/tag/v0.12.9

**Comment by cwRichardKim** [comment]
@ivpusic any way to access this from rn 0.29?

**Comment by habovh** [comment]
@cwRichardKim you could fork the repo to a point in time where it was RN 0.29 compatible, and do the same changes that were done in [PR #282](https://github.com/ivpusic/react-native-image-crop-picker/pull/282/files).

**Comment by cwRichardKim** [comment]
@habovh ended up upgrading to .42. it was a nightmare, but finally managed to get it to work

**Comment by magrinj** [comment]
If you use cocoa pods and you try to build it in release mode, you're going to get a failed compilation, because you need to change this in you Podfile (until it will be released on cocoapods):
From
```
pod 'RSKImageCropper'
```
To
```
pod 'RSKImageCropper', :git => 'https://github.com/ruslanskorb/RSKImageCropper.git', :branch => 'master'
```

**Comment by tedjames** [comment]
I'm having trouble finding the "RSKImageCropper.xcodeproj/project.pbxproj" file.

I assume that this file should be found when I open RSKImageCropper.xcodeproj in Xcode but I cannot seem to find it anywhere... Any suggestions?

**Comment by kornerr** [comment]
RSKImageCropper 1.6.1 has the issue fixed and works fine with Xcode 8.3.1
